server.js

```diff
@@ -184,12 +214,47 @@ const authenticateUser = async (req, res, next) => {
   }
 };
 
-app.get('/loginmessage', authenticateUser);
-app.get('/loginmessage', (req, res) => {
-  res.json({secret: 'Login works'})
+
+// Add plant to garden
+app.post("/addplant", authenticateUser);
+app.post("/addplant", async (req, res) => {
+  try {
+  const { plantname, species } = req.body;
+  const accessToken = req.header("Authorization");
+  const user = await User.findOne({accessToken: accessToken});
+  const plants = await new Plant({
+    plantname: plantname,
+    species: species,
+    user: user._id
```

Comment:
Datapoints we had discusse per each plant included - Insert plant name
- Choose plant species
- Birthday
- upload plant picture
- Last watered
- Last soil change